src/models/generated/Source.jl

```diff
@@ -6,8 +6,8 @@ This file is auto-generated. Do not edit.
         name::String
         available::Bool
         bus::Bus
-        V_R::Float64
-        V_I::Float64
+        activepower::Float64
```

Comment:
This change will require to properly set-up new equations in LITS for Source devices.